web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    Component,
+    OnInit,
+    OnDestroy,
+    ChangeDetectionStrategy,
+    Output,
+    EventEmitter
+} from '@angular/core';
+
+import {
+    UrlRouteManagerService
+} from 'app/shared/services';
+import { UrlPath } from 'app/shared/models';
+
+@Component({
+    selector: 'pp-transaction-id-search-container',
+    templateUrl: './transaction-id-search-container.component.html',
+    styleUrls: ['./transaction-id-search-container.component.css'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TransactionIdSearchContainerComponent implements OnInit, OnDestroy {
```

Comment:
Remove unimplemented lifecycle hook
```suggestion
export class TransactionIdSearchContainerComponent implements OnInit {
```